portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -28,11 +28,13 @@
 @WebServlet("/delete-data")
 public class DeleteCommentServlet extends HttpServlet {
 
+  private static final String ENTITY_KIND = "Comment";
```

Comment:
nit: prefer this ENTITY_KIND string to be in a common file that can be resused by both `DeleteCommentServlet` and `DataServlet` since from my understanding, they have to be equivalent, but can be done in separate PR, or add TODO to work on it later.